cli/src/verify.ts

```diff
@@ -0,0 +1,20 @@
+import { CircuitScaffold } from "./scaffold";
+import { getFunctionFromTs, getRunCircuitFromTs, getUint8ArrayFromBuffer, readJsonFromFile } from "./utils";
+
+export const verify = async (path: string, options: { vk: string, proof: string, instances: string, circuit: string }) => {
```

Comment:
To verify shouldn't you only need: `vkey`, `instances`, `proof`? There should no longer be a need for `circuit`

Reply:
reading the vkey seems to require `BaseCircuitParams`, which is instantiated in `halo2-wasm` from `CircuitConfig`. Currently the circuit file is responsible for exporting the circuit config (https://github.com/axiom-crypto/halo2-browser/blob/8737bac8ce42c2f5c9135f5f9daf482b15bbcc82/cli/src/examples/circuit.ts#L20). Will add this to the readme.